app/code/Magento/Sales/Model/Order/Creditmemo.php

```diff
@@ -1525,5 +1539,12 @@ public function setExtensionAttributes(\Magento\Sales\Api\Data\CreditmemoExtensi
         return $this->_setExtensionAttributes($extensionAttributes);
     }
 
+    /**
+     * @return InvoiceRepositoryInterface|mixed
+     * @deprecated
+     */
+    private function getInvoiceRepository() {
+        return $this->invoiceRepository ?: ObjectManager::getInstance()->get(InvoiceRepositoryInterface::class);
```

Comment:
Please avoid such methods creation and move this logic to constructor.

Reply:
Okay